### DISCUSSION TITLE
Welcome to openintegrationhub Discussions!

### DISCUSSION THREAD

**Discussion by RobinBrinkmann** [openintegrationhub/openintegrationhub#1135] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community on topics that are not worth opening an issues right away. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

For more in-depth and personal exchange, please also join our [Slack Channel](https://join.slack.com/t/openintegrationhub/shared_invite/zt-mba97vn9-xus3ZbVxnMr2oQwGegIk5Q)

**Comment by achimbecker-aventurin** [comment]
I'm trying to understand where I can find a more "consolidated state of things" view in terms of ready for productive usage. I'm consulting an SME company in Germany, which is in need of such an integration solution. But I do not find any current information other than those from 2017/18 where the press was covering the early start and funding through BMWI. Can you point me to a place where I can assess the ready-for-prime-time question? Number of active installation, number of productive customers, references, throughput numbers, ...
best, Achim

**Reply by RobinBrinkmann** [reply]
Hi @achimbecker-aventurin

thanks for reaching out. The best place to get in touch is the above mentioned slack-channel, but I am also happy to arrange a direct chat - robin.brinkmann@cloudecosystem.org.

Some quick points to your questions:
- There is a number of companies and products utilising the framework and it is ready for productive use
- It is also under active development, with renewed funding by the BMWi
- Open Integration Hub is first and foremost a framework, so the code you find here on github, and not a managed product in itself
- You can take the code, deploy it and integrate it in your existing solution, build a new one on top or just run some flows - it is up to you
- I would advise starting with [a local deployment](https://openintegrationhub.github.io/docs/3%20-%20GettingStarted/Intro.html) to see if it fits your needs
- If you are more interested in a managed service, we are currently working on a "aaS" version of the framework, which can  be tested very shortly
